NuGet/ChustaSoft.Tools.Authorization/Services/UserService.cs

```diff
@@ -215,6 +225,37 @@ public void Review(TUser user)
                 user.Culture = _authorizationSettings.DefaultCulture;
         }
 
+        public async Task<SignInResult> ExternalSignInAsync(bool isPersistent)
+        {
+            var loginInfo = await _signInManager.GetExternalLoginInfoAsync();
+            return await _signInManager.ExternalLoginSignInAsync(loginInfo.LoginProvider, loginInfo.ProviderKey, isPersistent);
+        }
+
+        public async Task CreateExternalAsync(string defaultRole)
```

Comment:
defaultRole is not required to be sent as parameter. UserService has the settings DTO injected as well.

Reply:
Getting the role from the settings